baseline/OSPS-AC.yaml

```
@@ -50,7 +50,9 @@ criteria:
       the project's repository by limiting the
       permissions granted to collaborators.
     details: |
-      Configure the project's version control
+      Most public version control systems are configured
+      in this manner.  If using a custom sytsem,
+      configure the project's version control
       system to assign the lowest available
```

```suggestion
      in this manner. Ensure the project's version control
      system always assigns the lowest available
```

Fixing a typo and changing the verbiage a bit.